bigcode_eval/evaluator.py

```diff
@@ -42,7 +42,7 @@ def __init__(self, accelerator, model, tokenizer, args):
 
     def generate_text(self, task_name, intermediate_generations=None):
         task = tasks.get_task(task_name, self.args)
-        dataset = task.get_dataset()
+        dataset = task.get_dataset(self.args)
```

Comment:
Can you make the arguments here more explicit? It'll make it easier to confirm what's supported from the signature.